kura/org.eclipse.kura.core.configuration.test/.gitignore

```diff
@@ -0,0 +1 @@
+/bin/
```

Comment:
@cdealti I believe that the deployed ConfigurationService(Impl) would have to be retrieved from the OSGi container for it to be what you wanted from #785. Here it is instead instantiated and manually injected with dependencies as in my unit tests.